src/app/components/utils/htmlutils.ts

```diff
@@ -0,0 +1,11 @@
+export class HTMLUtils {
```

Comment:
I don't know the rest of the code but in my opinion the `utils` folder should be outside `components` folder.

Reply:
I just followed the current folder structure.

Reply:
Hi,

Since there isn't much code, you can remove the HTML Utils and check the tagname inside the accordion.ts.

Reply:
Sure. Changes are made now :)